critiquebrainz/frontend/external/musicbrainz_db/label.py

```diff
@@ -0,0 +1,23 @@
+from brainzutils import cache
+from brainzutils.musicbrainz_db.label import fetch_multiple_labels
+from critiquebrainz.frontend.external.musicbrainz_db import DEFAULT_CACHE_EXPIRATION
+from critiquebrainz.frontend.external.relationships import label as label_rel
+
+
+def get_label_by_id(mbid):
+    """Get label with MusicBrainz ID.
+
+    Args:
+        mbid (uuid): MBID(gid) of the label.
+    Returns:
+        Dictionary containing the label information
+    """
+    key = cache.gen_key(mbid)
+    label = cache.get(key)
+    if not label:
+        label = fetch_multiple_labels(
+            [mbid],
+            includes=['artist-rels', 'url-rels'],
+        ).get(mbid)
+        cache.set(key=key, val=label, time=DEFAULT_CACHE_EXPIRATION)
+    return label_rel.process(label)
```

Comment:
Do you have the complete musicbrainz_db imported in your local setup or just the sample dumps? (I am curious if the same `process` works for labels? We will find out when this is deployed to beta, but please keep in mind as we have potential duplicated code we can reduce :)

Reply:
Actually, let's try to remove duplication of this code in a separate PR in some way. (I'll create a ticket for this)

Reply:
I was looking at this and I think it can be simplified. I believe I manually downloaded and imported the dumps.

Reply:
:+1: